### PR TITLE
[modular][ready] replaces a bunch of onmob sprites with better bee versions

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

title
its basically 90% of https://github.com/unitystation/unitystation/pull/7108
but not for unity, check there for exact PR links and all that

because my catgirl slave master told me to:
image
image
image
image
image (director one was skipped because I didn't realise)
image (altchef was skipped because this isn't CM and I couldn't be arsed)
image
image

## Why It's Good For The Game

they're better
FAQ's
Q: Why is my game crashing on startup?
A: You didn't install CSS

Q: Why just the OnMob? There are obj files there as well!
A: ~~I'm lazy, make another PR for that.~~ Nope, here now.

Q: Is this mod part of the Stubborn Series.
A: I guess.

Q: Can I bring companions?
A: What?

Q: Why is this pr so short?
A: I made it in a few hours and most of that was "hmm delete the dupe, point the aest folder to the right place."

Q: Will you add radiant missions?
A: What? This is a clothing pr.

Q: Why are the sprites so bad?
A: Uninstall and un-endorse.

Q: This PR conflicts with [INSERT ANY NUMBER OF NUDE OR NON-STUBBORN APPROVED CLOTHING PRS]. What should I do?
A: Uninstall and un-endorse. This PR is not worth fretting over conflicts, really.

Q: Is there even a plot? What about characterization? Basic roleplaying? Freedom of choice?
A: Uninstall and un-endorse.

Q: I want to find EVERY INCH of content - where is it?
A: Click the file tab.

Q: How often will you update this PR?
A: If there are errors, otherwise, not much.

Q: Will you implement [INSERT ANY NUMBER OF IMPLAUSIBLE FEATURES THAT REQUIRES LABOURIOUS IMPLEMENTATION]?
A: Probably not.

Q: Why does this PR have[INSERT PEDANTIC RANT OVER POSSIBLE UNDERMINING OF THE SACRED CANON OF SKYRAT]?
A: Uninstall and un-endorse.

Q: This PR sucks. Can't you [INSERT PETULANT HARANGUE]?
A: Uninstall and un-endorse.

Q: This looks like a sign of regression and a steady, pathetic slide into drunken indolence. What have you got to say for yourself?
A: Uninstall and un-endorse.

Q: Will the next clothing PR be this bad?
A: Uninstall and un-endorse.

## Changelog
:cl: Plateaguy2000 (who done basically nothing), Naevii, ivanmixo, Agameofscones and SomeAngryMiner (who done all the actual work)

expansion: replaced a bunch of onmob sprites with better bee versions
imageadd: new shiny onmob sprites from bee
imagedel: old sprites in our modular folder
/:cl:

### CODE REVIEW

**Comment by tf-4** [comment]
Put screenshots of sprite changes in the PR body

**Comment by Arctaisia** [comment]
Would be nicer if we got the roboticist jumpsuits as an alternative jumpsuit, as every other roboticist thing we have is black-red, not black-purple.

**Comment by thestubborn** [comment]
> 
> 
> Put screenshots of sprite changes in the PR body

put your face on my ass

but seriously, gimme a sec

**Comment by thestubborn** [comment]
> 
> 
> Would be nicer if we got the roboticist jumpsuits as an alternative jumpsuit, as every other roboticist thing we have is black-red, not black-purple.

it do kinda be more drippy....
but ask gandalf, I just went "cool suit" and he went "port all of it"

File: modular_skyrat/modules/aesthetics/clothing/clothing.dm [hunk 1]
```diff
@@ -1,3 +1,72 @@
+
+///////////////////////////////////////////////////////////////////////SHOES
+
+/obj/item/clothing/shoes/sneakers/white
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/aesthetics/clothing/clothing.dm]: Remove that, sneakers are using GAGS so that will just do nothing.

Comment (thestubborn) [reply modular_skyrat/modules/aesthetics/clothing/clothing.dm]: fiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine

**Comment by thestubborn** [comment]
see above for alternates

because i'm probably done again
thanks golden!

**Comment by thestubborn** [comment]
and below

**Comment by Sethnov** [comment]
any digitigrade versions?

**Comment by thestubborn** [comment]
> 
> 
> any digitigrade versions?

coats yes, the uniforms, not so much, feel free to like
I dunno, craft em or whatever and I'll add em, but I ain't really gonna fret over em myself

**Comment by Gandalf2k15** [comment]
I'm not dealing with the conflicts of the others. This one will do.

**Pull request merged**